File: Wildcamper.API/Models/Rating.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Wildcamper.API.Models
{
  public partial class Rating
  {
    public int RatingId { get; set; }
    public int PlaceId { get; set; }
    public string CreatorId { get; set; }
    public string Comment { get; set; }
    public int Stars { get; set; }
    public DateTime CreatedDate { get; set; }
    public DateTime ModifiedDate { get; set; }

    [JsonIgnore]
    public virtual User Creator { get; set; }

    [JsonIgnore]
    public virtual Place Place { get; set; }
  }
}

File: WildcamperMobile/lib/Domain/model/Rating.dart
import 'package:WildcamperMobile/Data/DataAccess/DTO/RatingDto.dart';

class Rating {
  int ratingId;
  String creatorId;
  int placeId;
  String comment;
  int stars;

  Rating({
    this.ratingId,
    this.creatorId,
    this.placeId,
    this.comment,
    this.stars,
  });

  factory Rating.fromDto(RatingDto rtg) => Rating(
      ratingId: rtg.ratingId,
      creatorId: rtg.creatorId,
      comment: rtg.comment,
      placeId: rtg.placeId,
      stars: rtg.stars);
}
